### DISCUSSION TITLE
Every GridSearchCV returns score=nan

### DISCUSSION THREAD

**Discussion by ghost** [scikit-multilearn/scikit-multilearn#303] [Q&A]
Hi Colleagues,

By using the example GridSearchCV as explained in the examples, I always get score=nan for any scoring (accuracy, f1_score macro) etc.

Any ideas? Anything wrong I am doing?
Thanks

Here's an example code:

```
from skmultilearn.problem_transform import BinaryRelevance
from sklearn.model_selection import GridSearchCV
from sklearn.naive_bayes import MultinomialNB
from sklearn.svm import SVC

df_train = pd.read_csv("../../Data Cleansing/DataCleansingResults/df_train_organ_each_with_zeroes_tissue.csv", sep=",")
df_train = df_train.drop(['Unnamed: 0'], axis=1)
print(df_train.head())

X = df_train.iloc[:, :4].values
print(X)
print(X.shape, type(X))

y = df_train.iloc[:, 4:].values
print(y)
print(y.shape, type(y))

parameters = [
    {
        'classifier': [MultinomialNB()],
        'classifier__alpha': [0.7, 1.0],
    },
    {
        'classifier': [SVC()],
        'classifier__kernel': ['rbf', 'linear'],
    },
]

clf = GridSearchCV(BinaryRelevance(), parameters, scoring='f1_macro', verbose=4)
clf.fit(X, y)

print (clf.best_params_, clf.best_score_)
```
